bsip-0083.md

```diff
@@ -0,0 +1,74 @@
+
+Governance is an important component of the core BTS token and effective application of governance throughout the BitShares protocol is imperative to the long-term success of the project.  A number of BSIPs have addressed the shortcomings of the existing vote calculation mechanism. This BSIP changes vote calculation which currently confers governance rights which are intended for assignment to the protocol.
+
+In short, this BSIP will put the BitAssets under the control of a designated user account. With this change, the newly appointed BitAssets manager will be able to independently maintain the parameters of their derivative asset without interference from the platform’s block producers and committee members. At the same time, the platform itself and its committee will not be unduly influenced by the demands of private product management. An advantage gained by this is that the BitAsset manager will be able to react appropriately to rapidly changing market conditions, as may be required to properly maintain a successful derivative asset.
+
+# Rationale
+Because the BitAssets are currently controlled by platform-wide governance mechanisms, and because these are not well suited to the management of smart coins, which may at times need to respond quickly to market conditions, the authors propose to reorganize these assets to utilize a directly appointed multisignature authority and thereby enable these assets to update in synchronization with market ebbs and flows.
+
+The authors recommend reassigning the blockchain-controlled BitAssets’ issuer field from committee-account to cn-vote, which is a multisignature account composed of seven different parties and having numerous proxy followers, and setting cn-vote as the sole feed producer, until cn-vote can delegate this role to accounts of its own appointment. The authors selected cn-vote for this role because cn-vote has demonstrated an active and vested interest in the stability and viability of the BitAsset products. Should cn-vote accept this appointment, the specification of the BitAsset parameters and feed production will become the responsibility of the cn-vote multisignature account.
+
+As a result of this assignment, the block producers will no longer provide feeds for BitAssets, and the new set of feed producers will no longer be affected by direct stakeholder voting.
```

Comment:
Which bitassets are in scope of this BSIP? Just bitCNY? What about the following assets owned by the committee-account? GAS, DIESEL, KRW, BTC, OIL, SILVER, GOLD, TRY, SGD, HKD, RUB, SEK, NZD, CNY, MXN, CAD, CHF, AUD, GBP, JPY, EUR, USD, CROWDFUN, DRAGON, SERVICE, BTCSHA, TESTME, BANCOR, NASDAQC, HANGSENG, NIKKEI, SHENZHEN, SHANGHAI, ARS, RUBLE, XCD, RUPEES, FORINT

Reply:
Yes, all BitAssets are in scope of this BSIP; all other SmartCoins are out of scope. The specification section must be updated to note all of them. @oxarbitrage I believe you have the list and can update this?

Reply:
The account will continue to exist and will continue to be the owner of the SmartCoins. It is in the responsibility of the asset owner to care for the asset, which can include handing it over to someone else.

Reply:
> 
> 
> The account will continue to exist and will continue to be the owner of the SmartCoins. It is in the responsibility of the asset owner to care for the asset, which can include handing it over to someone else.

What you said is meaningless, and what you said is shirking the responsibility to a third party.

Reply:
Replaced cn-vote.